### PR TITLE
Add unit tests for org.apache.rocketmq.common.UtilAll

### PR DESCRIPTION
**Pull request by Braavos6**
I've analysed your codebase and noticed that `org.apache.rocketmq.common.UtilAll` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important.

### CODE REVIEW

**Review by paulkennethkent — APPROVED** [review]

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+        PowerMockito.mockStatic(System.class);
+        PowerMockito.when(System.currentTimeMillis())
+                .thenReturn(1558787014000l);
+
+        Assert.assertEquals("20190525132334000",
+                UtilAll.timeMillisToHumanString());
+    }
+
+    @Test
+    public void testTimeMillisToHumanStringLong2() {
```
Comment (petersear) [inline common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Why the "2" on the end of the test method name? I didn't see a testTimeMillisToHumanStringLong()

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+    }
+
+    @Rule
+    private TemporaryFolder folder = new TemporaryFolder();
+
+    @Test
+    public void testGetDiskPartitionSpaceUsedPercent1() throws IOException {
+        File file = folder.newFile("foo.txt");
+
+        Assert.assertNotNull(UtilAll
```
Comment (petersear) [inline common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Better to break the line at `(`?

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+    @Test
+    public void testAsLong() {
+        Assert.assertEquals(1l, UtilAll.asLong("foo", 1l));
+        Assert.assertEquals(1556195014000l,
+                UtilAll.asLong("1556195014000", 1l));
+    }
+
+    @Test
+    public void testFormatDate() {
+        Assert.assertEquals("2019-04-25 13:23:34", UtilAll
```
Comment (petersear) [inline common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Maybe break the line after `UtilAll.formatDate(`?

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java [hunk 4]
```diff
@@ -0,0 +1,327 @@
...
+                .formatDate(new Date(1556195014000l), "yyyy-MM-dd HH:mm:ss"));
+    }
+
+    @Test
+    public void testParseDate() {
+        Assert.assertNull(UtilAll.parseDate("2019-04-25 13:23:34", ""));
+
+        Assert.assertEquals(new Date(1556195014000l), UtilAll
+                .parseDate("2019-04-25 13:23:34", "yyyy-MM-dd HH:mm:ss"));
```
Comment (petersear) [inline common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Maybe break line after `UtilAll.parseDate(`?

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java [hunk 5]
```diff
@@ -0,0 +1,327 @@
...
+    }
+
+    @Test
+    public void testIpToIPv4Str() {
+        Assert.assertEquals("10.1.2.3",
+                UtilAll.ipToIPv4Str(new byte[]{10, 1, 2, 3}));
+        Assert.assertEquals("172.16.2.3",
+                UtilAll.ipToIPv4Str(new byte[]{(byte) 172, 16, 2, 3}));
+        Assert.assertEquals("192.168.2.3",
+                UtilAll.ipToIPv4Str(new byte[]{(byte) 192, (byte) 168, 2, 3}));
```
Comment (petersear) [inline common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: line > 80 chars (81)

**Review by petersear — CHANGES_REQUESTED** [review]

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java (hunk 1 above)
Comment (Braavos6) [reply common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Done

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java (hunk 2 above)
Comment (Braavos6) [reply common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Done

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java (hunk 3 above)
Comment (Braavos6) [reply common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Done

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java (hunk 4 above)
Comment (Braavos6) [reply common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Done

File: common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java (hunk 5 above)
Comment (Braavos6) [reply common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: It is 80 chars for me.. I moved it tho

Comment (petersear) [reply common/src/test/java/org/apache/rocketmq/common/UtilAllTests.java]: Sorry, I double-checked and you were right.

**Review by petersear — APPROVED** [review]

**Pull request closed without merging**